.circleci/config.yml

```
@@ -390,15 +390,13 @@ commands:
               --timeout 5m
 
   publish-artifacts:
-    parameters:
-      artifact-type:
-        description: core or ui
-        type: string
     steps:
       - deploy:
           name: Upload Navigation SDK to SDK Registry
           command: |
-            make << parameters.artifact-type >>-upload-to-sdk-registry
+            export GITHUB_TOKEN=$(./mbx-ci github writer token)
```

```suggestion
            export GITHUB_TOKEN=$(./mbx-ci github writer public token)
```
Or I can update it in #5233 myself.

---

Happy to add it here.